src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.cpp

```diff
@@ -53,6 +53,16 @@ llvm::SmallVector<int64_t> createInt64VectorFromIndexExpr(
   return literalVector;
 }
 
+Value getTosaConstTensorSingleF32(
```

Comment:
This is already implemented in the `tosaBuilder` https://github.com/ljfitz/onnx-mlir/blob/7a1e59439f255f836e88f1677da6a0c8c5463ccb/src/Conversion/ONNXToTOSA/DialectBuilder.cpp#L92.